### ISSUE TITLE
Tag envmodules isn't recognized

### ISSUE THREAD

**Issue by otting** [JetBrains-Research/snakecharm#332]
envmodules worked just fine about 2 month ago, now the tag breaks all the highlighting below the tag...
![image](https://user-images.githubusercontent.com/5678451/96240663-79a5ea80-0fa1-11eb-9d72-cb46840f41b9.png)

**Comment by iromeo** [comment]
@otting thank you for reporting this. I'll try to release the updated plugin version in the second half of December.

Snakemake parsing engine treats section names as keywords, so you get similar parsing errors in case of the unsupported or misspelled section names. I'll add the `envmodules` word into the list of known section names and consider some better implementation to avoid similar issues in the future.

**Comment by iromeo** [comment]
Issue is caused by https://github.com/JetBrains-Research/snakecharm/issues/334

**Comment by iromeo** [comment]
> envmodules worked just fine about 2 month ago
I'm sure they didn't work properly 2 month ago, maybe there were a bug in missing highlighting

**Comment by otting** [comment]
> > envmodules worked just fine about 2 month ago
> 
> I'm sure they didn't work properly 2 months ago, maybe there was a bug in missing highlighting.

I don't remember seeing this behaviour when I first switched to pycharm with snakecharm, but I started with an almost done workflow so maybe I did just not look at it before.

**Comment by iromeo** [comment]
@otting Could `envmodules:` section argument contain wildcards, e.g.:
```
rule foo:
    input: "boo/{vers}/input"
    envmodules: "tool/{vers}/cmd"
```
will it work at runtime?

**Comment by otting** [comment]
@iromeo I don't know, sorry. I don't see a reason to do so but also no reason why it would not work.

**Comment by iromeo** [comment]
@otting Wildcards are allowed not in all snakemake sections, but it isn't documented. Also in some sections you have to use `{wildcards.name}` in others just `{name}`.  The easiest way is to check on runtime or to digg into snakemake source code. In the plugin I'm trying to adress it correctly and warn in case of incorrect usage. Ok, I'll try to find this in snakemake sources.

**Comment by iromeo** [comment]
Fixed, will be in the coming 0.9.0 release
